Vagrantfile

```diff
@@ -123,10 +123,14 @@ Vagrant.configure('2') do |config|
         fi
       done
       cd "$HOME"
-      nohup /opt/chefdk/embedded/bin/ruby -run -e httpd "$HOME/app_service" -p5000 2>&1 > /dev/null &
+      netstat -nl | grep :5000 > /dev/null || nohup /opt/chefdk/embedded/bin/ruby -run -e httpd "$HOME/app_service" -p5000 2>&1 > /dev/null &
```

Comment:
It looks to see if port 5000 is listening already. If it isn't then we attempt to start the ruby process that will listen on port 5000.

Basically this means that when we run `vagrant provision`, we don't try to start this ruby process over and over and over again.